src/shared/api/urls.ts

```diff
@@ -153,8 +156,13 @@ export function getProxyUrlIfNecessary(url:any) {
 
 export function getOncoKbApiUrl() {
     let url = AppConfig.serverConfig.oncokb_public_api_url;
-    return getProxyUrlIfNecessary(url);
+    if (typeof url === 'string') {
+        return buildCBioPortalAPIUrl(`proxy/${trimProtocol(url)}`);
+    } else {
+        return undefined;
```

Comment:
does calling code handle undefined?  I guess typescript would complain if it didn't, right?

Reply:
As far as I can tell, we inject these url to the client directly bypassing the constructer. It will have problem if url is undefined. But it seems like we will never run into this scenario since we have the serverConfigDefaults defined. @alisman